docs/DeveloperGuide.md

```diff
@@ -257,29 +257,53 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
-
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+* Uni students who are learning a new language
 
+**Value proposition**: 
+* Fast way to generate flashcards to learn new languages
+* Help students memorise words when learning a new language
+* Gamified features such as recording of scores and tracking of improvements to make learning engaging and encourage continued usage
 
 ### User stories
 
-Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
-
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
-
-*{More to be added}*
+Categories: Usage pattern, user behaviours, general user, multiple language user, user learning style, user learning 'rate', level of experience, user collaboration
+
+Category | As a/an... | I want to... | So that I...
```

Comment:
I think we should keep the `Priority` column, and place all the user stories with the features we are implementing at the top, while the rest can be low priority for now, e.g.

Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
Priority | Category | As a/an... | I want to... | So that I...
-------- | -------- | ---------- | ------------ | ----------------
*** | General user | user | be able to add new flash cards | 
*** | ... | ... | ... | ... 
\* | User learning style | auditory user | hear the vocabulary that I stored in the flashcard | can better memorize the words by sound
\* | ... | ... | ... | ...